challenge_help_me_carry/README.md

```diff
@@ -0,0 +1,38 @@
+# Challenge Help Me Carry 2017
+Responsible: Tim Assman
+
+## Test Log
+
+
+## Setup
+amigo1/sergio1:
+
+    astart/sstart # (if not already running)
+
+amigo2/sergio1:
+
+    ahardware/shardware: base, spindle, arms (left-arm is used) head
+
+amigo1/sergio1:
+
+    amiddle/smiddle
```

Comment:
According to the new launch structure, a challenge-specific launch file should be added to amigo_bringup.
Then, you can launch amigo-challenge-help-me-carry if you do it correctly :-)